packages/redux/src/recommendations/reducer.ts

```diff
@@ -73,23 +69,38 @@ const result = (
   }
 };
 
-export const getRecommendationsError = state => state.error;
-export const getAreRecommendationsLoading = state => state.isLoading;
-export const getRecommendations = state => state.result;
+export const getRecommendationsError = (state: State): State['error'] =>
+  state.error;
+export const getAreRecommendationsLoading = (
+  state: State,
+): State['isLoading'] => state.isLoading;
+export const getRecommendations = (
+  state: State | undefined,
+): State['result'] | undefined => state?.result;
+
+const reducers = combineReducers({
+  isLoading,
+  error,
+  result,
+});
 
 /**
- * Reducer for product recommendations state.
- *
- * @function recommendationsReducer
- * @static
+ * Reducer for forms state.
  *
- * @param {object} state - Current redux state.
- * @param {object} action - Action dispatched.
+ * @param state - Current redux state.
+ * @param action - Action dispatched.
  *
- * @returns {object} New state.
+ * @returns New state.
  */
-export default combineReducers({
-  error,
-  isLoading,
-  result,
-});
+const recommendationsReducer: ReducerSwitch<State, AnyAction> = (
+  state,
+  action = { type: undefined },
```

Comment:
Can you review this default parameter value to match the other reducers (forms namely)?

Reply:
Source updated, can you check again.
FormReducer, updated too.